scripts/openfdd_ci_babysit_pr.sh

```
@@ -0,0 +1,54 @@
+
+for ((i = 1; i <= MAX_WAKES; i++)); do
+  state="$(gh pr view "$PR" --repo "$REPO" --json state --jq '.state' 2>/dev/null || echo UNKNOWN)"
+  if [[ "$state" == "MERGED" ]]; then
+    log "PR already merged"
+    exit 0
+  fi
+
+  if pr_all_green; then
+    log "all checks green — merging PR #$PR"
+    gh pr merge "$PR" --repo "$REPO" --squash --delete-branch >>"$LOG" 2>&1 || true
+    sha="$(gh api repos/$REPO/commits/master --jq '.sha[0:7]' 2>/dev/null || echo unknown)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)